enrich-points-with-boundaries/geodatabase/src/test/scala/com/precisely/bigdata/sample/spark/VerificationDriver.scala

```diff
@@ -21,6 +21,15 @@ object VerificationDriver {
     val sparkConf = new SparkConf()
     sparkConf.setIfMissing("spark.master", "local[*]")
     sparkConf.set("spark.sql.autoBroadcastJoinThreshold", "-1")
+
+    val sparkMajorVersion=org.apache.spark.SPARK_VERSION.split('=')(0).split('.')(0).toInt
+    if(sparkMajorVersion >= 3){
+      val sparkMinorVersion=org.apache.spark.SPARK_VERSION.split('=')(0).split('.')(1).toInt
+      if(sparkMajorVersion == 3 && sparkMinorVersion == 0){
```

Comment:
Do we need to check sparkMajorVersion here again? We have already check majorVersion==3 in outer IF block

Reply:
I made changes in the code as advised.